### PR TITLE
Missing test data from the rename change

### PR DESCRIPTION
**Pull request by NonStatic2014**

### CODE REVIEW

**Comment by tmccrmck** [comment]
@pranavsharma @snnn is it okay to add a 26 kb file?

**Comment by snnn** [comment]
We already put the test data (including the minst model) in build dir, could you reuse it?

**Comment by NonStatic2014** [comment]
Make it download from blob storage for now. Will need to use test data in build dir after we merged to upstream.

File: onnxruntime/test/server/integration_tests/function_tests.py [hunk 1]
```diff
@@ -330,24 +341,29 @@ class HttpEndpointTests(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cmd = [cls.server_app_path, '--http_port', str(cls.server_port), '--model_path', os.path.join(cls.model_path, 'mnist.onnx'), '--log_level', cls.log_level]
-        print('Launching server app: [{0}]'.format(' '.join(cmd)))
+        onnx_model = os.path.join(cls.model_path, 'mnist.onnx')
```
Comment (tmccrmck) [inline onnxruntime/test/server/integration_tests/function_tests.py:337]: A lot of code is duplicated here. Perhaps pull at least the URL to one single location to make it easy to change

**Review by tmccrmck — APPROVED** [review]

**Pull request merged**